Makefile

```diff
@@ -48,8 +48,11 @@ requirements: ## Install requirements for development
 requirements-test: ## Install requirements for testing
 	${VENV_BIN}/pip install -qr requirements/test.txt --exists-action w
 
-migrate:  ## Apply database migrations
-	${VENV_BIN}/python manage.py migrate --no-input
+production-requirements:
+	pip install -r requirements.txt --exists-action w
+
+migrate:
+	python manage.py migrate
```

Comment:
Any specific reason we want to omit the `--no-input`?

Reply:
It was an oversight. I've included it now.